safe/report/expressions/map_report.py

```diff
@@ -101,20 +110,40 @@ def distance_to_nearest_place(feature, parent):
     project_context_scope = QgsExpressionContextUtils.projectScope()
     registry = QgsMapLayerRegistry.instance()
 
+    active_layer = iface.activeLayer()
```

Comment:
So we don't use project variable?

We start having different behaviors like having an active layer in the legend and/or having project variables.

Can we select any layer in the multiexposure group, it will work? (whatever the outlayer selected in the legend, it will find the correct layer right?)